samples/Salesforce.gs

```diff
@@ -0,0 +1,78 @@
+/**
+ * Saleforce's Auth flow https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/quickstart_oauth.htm
+ */
+
+var CLIENT_ID = '...';
+var CLIENT_SECRET = '...';
+
+/**
+ * Authorizes and makes a request to the Saleforce API.
+ */
+function run() {
+  var service = getService();
+  if (service.hasAccess()) {
+    // GET requests require access_token parameter
+
+    var response = UrlFetchApp.fetch(service.getToken().instance_url + '/services/data/v24.0/chatter/users/me', {
```

Comment:
Nitpick: Perhaps define the URL on a separate line for clarity:

var url = service.getToken().instance_url + '/services/data/v24.0/chatter/users/me';
var response = UrlFetchApp.fetch(url, ...

Reply:
@jordan-cook , can you do this? Thanks.